client/src/app/ui/modules/prompt-checkbox/components/prompt-checkbox/prompt-checkbox.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { MAT_CHECKBOX_DEFAULT_OPTIONS, MatCheckboxDefaultOptions } from '@angular/material/checkbox';
+import { marker as _ } from '@biesbjerg/ngx-translate-extract-marker';
+
+import { PromptService } from '../../../prompt-dialog';
+
+@Component({
+    selector: `os-prompt-checkbox`,
+    templateUrl: `./prompt-checkbox.component.html`,
+    styleUrls: [`./prompt-checkbox.component.scss`],
+    providers: [
+        { provide: MAT_CHECKBOX_DEFAULT_OPTIONS, useValue: { clickAction: `noop` } as MatCheckboxDefaultOptions }
+    ]
+})
+export class PromptCheckboxComponent {
+    @Input() public checked: boolean;
+    @Input() public disabled: boolean;
+    @Output() selected = new EventEmitter<boolean>();
```

Comment:
It is good practice to add a short description to public members especially for components that should be reusable. 

```suggestion
    /**
      * Indicates if the checkbox is checked
      */
    @Input() 
    public checked: boolean;
    
    /**
      * ...
      */
    @Input() 
    public disabled: boolean;
    
    /**
      * ...
      */
    @Output() 
    selected = new EventEmitter<boolean>();
```